File: RP6/d5-2.cpp
#include <iostream>
using namespace std;

double celsius_para_fahrenheit(double celsius) {
    return (celsius * 9/5) + 32;
}

int main() {
    double celsius;

    cout << "Digite a temperatura em Celsius: ";
    cin >> celsius;

    double fahrenheit = celsius_para_fahrenheit(celsius);

    cout << "Temperatura em Fahrenheit: " << fahrenheit << endl;

    return 0;
}

File: RP6/d5-2.java
import java.util.Scanner;

public class Ex5_V2 {

    public static double celsius_para_fahrenheit(double celsius) {
        return (celsius * 9/5) + 32;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double celsius;
        System.out.print("Digite a temperatura em Celsius: ");
        celsius = sc.nextDouble();

        double fahrenheit = celsius_para_fahrenheit(celsius);

        System.out.println("Temperatura em Fahrenheit: " + fahrenheit);
    }
}